services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
```

Comment:
check [README.md](https://github.com/mate-academy/py-tickets-and-orders#readme) what function supposed to return